github/Repository.py

```diff
@@ -2190,6 +2385,35 @@ def create_file(
             ),
         }
 
+    def get_repository_advisories(
```

Comment:
Here, `repository` could also be removed, but on the other hand, it returns a `RepositoryAdvisory.

Reply:
So does the one above. Which naming convention would you like to see used here?

Reply:
Unless you have a strong preference, I think we should opt to use the naming convention in the GitHub API docs here.